components/push/src/lib.rs

```diff
@@ -183,20 +183,17 @@
 uniffi::include_scaffolding!("push");
 // All implementation detail lives in the `internal` module
 mod internal;
+use std::fmt::Display;
 use std::sync::Mutex;
 mod error;
 use error::*;
 
-// The following are only exposed for use by the examples
-pub use error::Result as InternalResult;
```

Comment:
The example now uses the high-level `PushManager` itself! and no longer needs access to internal values 😄